### PR TITLE
[Syntax file HTML]: Add WAI-ARIA htmlArg keywords

### PR DESCRIPTION
**Pull request by jorgesumle**
It Follows the [W3C Recommendation](https://www.w3.org/TR/wai-aria/)

### CODE REVIEW

**Comment by jorgesumle** [comment]
@brammool, @k-takata, someone can you please review this pull request? It's just a trivial update to a syntax file, so it won't take much time.

File: runtime/syntax/html.vim [hunk 1]
```diff
@@ -3,8 +3,8 @@
 " Maintainer:           Jorge Maldonado Ventura <jorgesumle@freakspot.net>
 " Previous Maintainer:  Claudio Fleiner <claudio@fleiner.com>
 " Repository:           https://notabug.org/jorgesumle/vim-html-syntax
-" Last Change:          2017 Dec 16
-" Included patch from Jorge Maldonado Ventura to add the dialog element
```
Comment (chrisbra) [inline runtime/syntax/html.vim:7]: why was this changed?

**Comment by chrisbra** [comment]
since the syntax file is basically unmaintained and you seem to have interest in it, would you volunteer to become the new maintainer?

**Comment by jorgesumle** [comment]
> since the syntax file is basically unmaintained and you seem to have interest in it, would you volunteer to become the new maintainer?

I'm already the maintainer.

**Comment by chrisbra** [comment]
> I'm already the maintainer.

Oh, I see. Well, the prefered way to update runtime files is to send the complete file to Bram for inclusion with Vim. Bram will then distribute it with the next runtime file update.

**Comment by jorgesumle** [comment]
@chrisbra, I see you already accepted the patch (https://github.com/vim/vim/commit/d2f3a8b8787333abf2300d38836b196955f10c00), but you didn't close this issue. Sorry for the confusion.
